### DISCUSSION TITLE
seo = more users + contributers

### DISCUSSION THREAD

**Discussion by jtkeyva** [ertgrulll/advstory#17] [Ideas]
i was thinking if you added popular app names to the description or summary, seo will help get more people seeing, using and contributing to advisory.

something like:
advisory provides you with instagram, snapchat and tiktok style stories (...that you can fully customize.....)

i can't tell you how many times i've searched google and pub.dev for instagram flutter, snapchat instagram, tiktok etc hoping to find something with similar functionality.

**Comment by ertgrulll** [comment]
Makes sense, I can add some popular keywords, thanks for the great advice 💯 Please fork the repository and change the readme, I'll merge your fork. So you can be a contributor of _AdvStory_, I couldn't do this manually.

**Reply by jtkeyva** [reply]
I sent a PR with an updated readme.
